src/hyperlinks.rst.include

```diff
@@ -1,25 +1,37 @@
 .. This document contains hyperlink references to be used throughout the
    documentation.
 
-.. _coreutils: https://www.gnu.org/software/coreutils/coreutils.html
-.. _Cylc: https://cylc.github.io/
+.. _Apollo Client: https://www.apollographql.com/docs/react/
+.. _Configurable HTTP Proxy: https://github.com/AbdealiJK/configurable-http-proxy
+.. _CurveZMQ: http://curvezmq.org/page:read-the-docs
 .. _Cylc User Guide: https://cylc.github.io/cylc-doc/latest/html/index.html
 .. _Cylc Workflow Design Guide: https://cylc.github.io/cylc-doc/latest/html/workflow-design-guide/index.html
+.. _Cylc: https://cylc.github.io/
 .. _DataPoint: https://www.metoffice.gov.uk/services/data/datapoint
 .. _EmPy: http://www.alcyone.com/software/empy/
-.. _extglob pattern matching: https://www.gnu.org/software/bash/manual/html_node/Pattern-Matching.html#Pattern-Matching
-.. _FCM: https://metomi.github.io/fcm/doc/
 .. _FCM User Guide: http://metomi.github.io/fcm/doc/user_guide/
+.. _FCM: https://metomi.github.io/fcm/doc/
 .. _INI: https://en.wikipedia.org/wiki/INI_file
 .. _Jinja2: https://jinja.palletsprojects.com/
-.. _metomi-vms: https://github.com/metomi/metomi-vms
+.. _Jupyter Hub: https://jupyterhub.readthedocs.io/en/stable/
+.. _Jupyter Lab: https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html
+.. _Jupyter Server: https://jupyter-server.readthedocs.io/en/latest/index.html
+.. _Protobuf: https://developers.google.com/protocol-buffers/
 .. _PyGTK: http://www.pygtk.org/
-.. _Python: https://www.python.org/
 .. _Python Regular Expressions: https://docs.python.org/2/library/re.html#regular-expression-syntax
+.. _Python: https://www.python.org/
 .. _Rose Documentation: https://metomi.github.io/rose
 .. _Rose GitHub: https://github.com/metomi/rose
 .. _SRS: https://code.metoffice.gov.uk/
 .. _SVN: https://subversion.apache.org/
+.. _Websocket: https://en.wikipedia.org/wiki/WebSocket
+.. _ZMQ: https://zeromq.org/
+.. _coreutils: https://www.gnu.org/software/coreutils/coreutils.html
+.. _extglob pattern matching: https://www.gnu.org/software/bash/manual/html_node/Pattern-Matching.html#Pattern-Matching
+.. _metomi-vms: https://github.com/metomi/metomi-vms
 
 .. _Cylc Flow: https://github.com/cylc/cylc-flow
+.. _Cylc UI: https://github.com/cylc/cylc-ui
+.. _UI: https://github.com/cylc/cylc-ui
```

Comment:
Yes, this is to allow us to reference "the UI" rather than "the Cylc UI".

"UI" is unused, however, "UI Server" is used so I added "UI" for completeness. I didn't add "Flow" as we would say "the Scheduler" rather than "the Flow", especially as "flow" (lower case) has a subtly different meaning now!